algorithmic_efficiency/workloads/librispeech_conformer/librispeech_jax/workload.py

```diff
@@ -166,10 +166,11 @@ def loss_fn(
                                        logit_paddings,
                                        targets,
                                        target_paddings)
-    # mask_batch is assumed to be shape [batch].
+    # mask_batch is assumed to be shape [batch] for other workloads
+    # but in speech cases it's [batch, target_len].
     if mask_batch is not None:
-      per_example_losses *= mask_batch
       mask_batch = jnp.logical_and(mask_batch, 1 - target_paddings)
```

Comment:
I think we multiplied by `mask_batch` after this line because `target_paddings` are already included in the ctc loss calculation, but I think it should not make a difference?